db/batchlog_manager.cc

```diff
@@ -11,6 +11,7 @@
  */
 
 #include <chrono>
+#include <exception>
```

Comment:
Is this include needed?

Reply:
clangd snuck it in and I left it be; strictly speaking it's probably redundant, but does it hurt?